tools/internal_ci/windows/grpc_distribtests_python.cfg

```diff
@@ -0,0 +1,26 @@
+# Copyright 2022 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Config file for the internal CI (in protobuf text format)
+
+# Location of the continuous shell script in repository.
+build_file: "grpc/tools/internal_ci/windows/grpc_distribtests_python.bat"
+timeout_mins: 120
```

Comment:
macOS's timeout is 240, should Windows's job match that?

https://github.com/grpc/grpc/pull/29141/files

Reply:
I copied the number from other windows distribtest, so not necessarily. These numbers are a bit random currently.